app/app/tests/api/api_v1/test_admins.py

```diff
@@ -60,11 +52,7 @@ def test_create_admin_existing_nonadmin(client: TestClient, superuser_token_head
 
 
 def test_create_admin_existing_student(client: TestClient, superuser_token_headers: dict, db: Session) -> None:
-    username = random_email()
-    # username = email
-    password = random_lower_string()
-    user_in = UserCreate(email=username, password=password, type="student")
-    user = crud.user.create(db, obj_in=user_in)
+    user = user = create_random_user(db, type="student")
```

Comment:
Extraneous `user = `

Reply:
eeeeeeek

Reply:
wao that happened in 2 places in the same file